core/src/util/tileID.h

```diff
@@ -16,6 +16,7 @@ struct TileID {
     
     TileID(int _x, int _y, int _z) : x(_x), y(_y), z(_z) {};
     TileID(const TileID& _rhs): x(_rhs.x), y(_rhs.y), z(_rhs.z) {};
+    TileID(const TileID&& _rhs): x(std::move(_rhs.x)), y(std::move(_rhs.y)), z(std::move(_rhs.z)) {};
```

Comment:
With all this discussion, I understood that I was wrong about std::move and stack values :(.
But because of this I read about std::forward and other stuff not related to this discussion (but related) :D. Thanks :D